local-volume/provisioner/pkg/discovery/discovery_test.go

```diff
@@ -56,6 +57,11 @@ var scMapping = map[string]common.MountConfig{
 	},
 }
 
+var testPVLabels = map[string]string{
```

Comment:
Also test labels that don't exist

Reply:
@msau42 Do you mean put some non standard labels in this map?  or do you mean specify some labels that don't exist on the node? If it is the latter, then the code that checks for that is currently in main() of main.go and not here. I can refactor the code there to make it unit test-able if that is what you had in mind.

Reply:
I mean labels that don't exist on the node.

Reply:
Fixed.